src/appleseed/renderer/modeling/object/object.h

```diff
@@ -110,6 +111,9 @@ class APPLESEED_DLLSYMBOL Object
     // Return the alpha map cached by on_frame_begin().
     const Source* get_alpha_map() const;
 
+    // Send this object to an object rasterizer.
+    virtual void rasterize(ObjectRasterizer& rasterizer) const;
```

Comment:
Maybe It would be clearer to introduce IRasterizable interface?
e.g.
class IRasterizable
{
  public:
    virtual void rasterize(ObjectRasterizer& rasterizer) const = 0
}

Reply:
Argh. If it was intended to skip object instances with no rasterize() implemented, please leave a comment here.

Reply:
I'm not sure to understand. As soon as `renderer::Object` has a `rasterize()` method, all objects (and thus all object instances) become rasterizable, although they might currently not yet override the default implementation which does nothing. We cannot use an `IRasterizable` interface to detect which objects implement it and which don't.

Reply:
Yes, I understood your intention later.